support/graphql.java.gen.build.gradle

```diff
@@ -0,0 +1,124 @@
+def VERSION_NAME = '0.2.1'
+
+buildscript {
+    repositories {
+        jcenter()
+    }
+}
+
+repositories {
+    jcenter()
+}
+
+apply plugin: 'java'
+apply plugin: 'maven'
+apply plugin: 'com.jfrog.bintray'
```

Comment:
Should this and the bintray block below be moved to the original `build.gradle` file?

Reply:
No, not necessary. We have the bintray artifacts already defined in the monorepo classpath. https://github.com/Shopify/android/blob/master/build.gradle#L447

Leaving this here allows us to publish from the monorepo, if we desire

Reply:
I'm more thinking for third party users of this lib?

Reply:
Well, if you don't do anything special, you'd never know there were two files. This is only valuable to us, in our monorepo configuration, because of the way we support custom gradle file names. For us to even use this, I'll have to open a PR pointing to the new file.